typescript/cli/src/tests/commands/core.ts

```diff
@@ -1,11 +1,45 @@
-import { $ } from 'zx';
+import { $, ProcessPromise } from 'zx';
 
 import { DerivedCoreConfig } from '@hyperlane-xyz/sdk';
+import { Address } from '@hyperlane-xyz/utils';
 
 import { readYamlOrJson } from '../../utils/files.js';
 
 import { ANVIL_KEY, REGISTRY_PATH } from './helpers.js';
 
+/**
+ * Deploys the Hyperlane core contracts to the specified chain using the provided config.
+ */
+export function hyperlaneCoreDeployRaw(
+  coreInputPath: string,
+  privateKey?: string,
+  skip?: boolean,
```

Comment:
Maybe give this a more clear name? In the context of a test, skip typically means to skip the test